lib/Config.js

```diff
@@ -234,8 +234,8 @@ class Config {
             }
             this.authData = authData;
         }
-        if (process.env.S3SPROXYD) {
-            data = process.env.S3SPROXYD;
+        if (process.env.S3DATA) {
```

Comment:
Careful with that change. I think our internal process uses `S3SPROXYD` so you will need to update there before merging here.